src/resources/goals.ts

```diff
@@ -0,0 +1,2 @@
+import resourceBase = require("./gen/goals_base");
```

Comment:
if you look at some of the sibling files, they manually add some fields to the resource which are displayed in the UI (`comment`/`properties`). they appear to be manually added so I will take another pass and do those if I have time.